packages/llm-bot/src/dispute-bot/DisputeDisputableRequests.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  DisputerStrategy,
+  OptimisticOracleClientV2,
+  OptimisticOracleClientV2FilterDisputeable,
+} from "../core/OptimisticOracleV2";
+import { Logger, MonitoringParams } from "./common";
+
+export async function disputeDisputableRequests(logger: typeof Logger, params: MonitoringParams): Promise<void> {
+  const oov2 = new OptimisticOracleClientV2(params.provider);
+
+  // Update the client with the latest block range.
+  const oov2ClientUpdated = await oov2.updateWithBlockRange();
+
+  const requests = Array.from(oov2ClientUpdated.requests.values());
+  const oov2FilterDisputable = new OptimisticOracleClientV2FilterDisputeable();
+
+  const filteredRequests = await oov2FilterDisputable.filter(requests);
+
+  const disputable = await Promise.all(filteredRequests.map(DisputerStrategy.process));
```

Comment:
can any of these calls error? i usually try to avoid processing things in a list like this since errors can prevent some from executing. if no errors are expected though this is fine